lib/definitions/length.js

```diff
@@ -40,6 +40,13 @@ imperial = {
     }
   , to_anchor: 1/12
   }
+, yd: {
+    name: {
+      singular: 'Yard'
+    , plural: 'Yards'  
+    }
+  , to_anchor: 1/0.3333
```

Comment:
Wouldn't the yards anchor be to feet? 1 foot = 3 yards so it seems like to_anchor should be equal to 3 here. That seems to product the correct results.